mmengine/visualization/visualizer.py

```diff
@@ -398,26 +415,68 @@ def draw_points(self,
         assert positions.shape[-1] == 2, (
             'The shape of `positions` should be (N, 2), '
             f'but got {positions.shape}')
-        colors = color_val_matplotlib(colors)  # type: ignore
-        self.ax_save.scatter(
-            positions[:, 0], positions[:, 1], c=colors, s=sizes, marker=marker)
+        if self.backend == 'matplotlib':
+            assert isinstance(marker, str) or marker is None, (
+                'The type of `marker` in `matplotlib` should be str, '
+                f'but got {type(marker)}')
```

Comment:
Please follow the google python style to raise the exeception

https://google.github.io/styleguide/pyguide.html#244-decision